data/forms/city/agentassignment.form

```diff
@@ -34,21 +34,13 @@
         <imagedepressed>BUTTON_SCROLL_DOWN_DEPRESSED</imagedepressed>
     </graphicbutton>
     <multilistbox id="AGENT_SELECT_BOX" scrollbarid="AGENT_SELECT_SCROLL">
-        <position x="16" y="32"/>
```

Comment:
Was this change intended to be in this commit?

If it's fixing a different issue (e.g. misaligned form), it probably makes sense to have a separate commit

Reply:
Hey Jonny offtopic but look pls i create PR wich readme)